src/Command/PermissionsMakeAdminCommand.php

```diff
@@ -51,7 +53,26 @@ protected function configure(): void
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $io = new SymfonyStyle($input, $output);
+
         $discordUserId = $input->getArgument('discord_user_id');
+        if (null === $discordUserId) {
+            /** @var UserEntity[] $allUsers */
+            $allUsers = $this->userEntityRepository->findAll();
+            $allUsersNames = array_map(function (UserEntity $x) {
+                return sprintf('%d (%s)', $x->getExternalId(), $x->getUsername());
+            }, $allUsers);
+
+            $helper = $this->getHelper('question');
+            $question = new ChoiceQuestion('Please select user from the list', $allUsersNames);
+
+            $response = $helper->ask($input, $output, $question);
+            if (!$response) {
```

Comment:
In which case `$response` can be null or empty? Is this check necessary?

Reply:
I assumed that ctrl+c, but maybe it just kills the app. Needs verification.